client/layout/index.jsx

```diff
@@ -117,6 +118,7 @@ Layout = React.createClass( {
 				{ config.isEnabled( 'keyboard-shortcuts' ) ? <KeyboardShortcutsMenu /> : null }
 				{ this.renderMasterbar() }
 				<div className={ loadingClass } ><PulsingDot active={ this.props.isLoading } chunkName={ this.props.chunkName } /></div>
+				{ this.props.connection === 'OFFLINE' && <OfflineStatus /> }
```

Comment:
I think we should have selectors for `isOnline / isOffline`. It gets quite repetitive otherwise. Or at the very least we should do the comparison in the connect function and just set the property to `isOffline`.